posthog/queries/test/test_trends.py

```diff
@@ -2742,6 +2742,61 @@ def test_breakdown_by_person_property_for_person_on_events(self):
                 if response["breakdown_value"] == "person3":
                     self.assertEqual(response["count"], 3)
 
+        def test_breakdown_by_person_property_for_person_on_events_with_zero_person_ids(self):
```

Comment:
hmm, this should've failed for person on events for now, not sure why it didn't on CI 🤔 . Will check.